### PR TITLE
Add examples from stubs to API, autocomplete, parameter help

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

File: src/editor/codemirror/themeExtensions.ts [hunk 1]
```diff
@@ -82,13 +82,13 @@ export const themeExtensions = (fontSize: string) => {
     },
     ".cm-tooltip.cm-completionInfo": {
       width: "20rem",
-      height: "10rem",
+      height: "10.5rem",
```
Comment (microbit-matt-hillsdon) [inline src/editor/codemirror/themeExtensions.ts:85]: Not confident in this hight. Might also look at the font size choices for the docs.

**Comment by microbit-robert** [comment]
Looks pretty good on my first pass. I found the following issues:

- `math.degrees()` - example doesn't use `degrees()`.
- `is_gesture("shake")` - use of double quotes.
- `display.scroll('micro:bit)` - missing closing single quote.
- `radio.send("hello")` - use of double quotes.
- `speech.sing()` - example doesn't use `sing()`.

**Comment by microbit-matt-hillsdon** [comment]
Thanks Rob. [eb21873](https://github.com/microbit-foundation/python-editor-next/pull/709/commits/eb21873f43f3922bd23d22b0c74c6798235e8538) fixes these and one other.

File: src/editor/codemirror/themeExtensions.ts (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply src/editor/codemirror/themeExtensions.ts:85]: I think this is OK having lived with it for a while.

File: src/editor/codemirror/language-server/documentation.ts [hunk 2]
```diff
@@ -95,8 +112,8 @@ export const wrapWithDocumentationButton = (
   docsAndActions.style.justifyContent = "space-between";
   docsAndActions.appendChild(child);
 
-  const button = docsAndActions.appendChild(document.createElement("button"));
-  button.innerHTML = `
+  const anchor = docsAndActions.appendChild(document.createElement("a"));
+  anchor.innerHTML = `
```
[the remark this answers is not in the document]
Comment (microbit-matt-hillsdon) [reply src/editor/codemirror/language-server/documentation.ts]: 664ad67f0784f5938e6e35b1a90e0f9ff6ca5882

**Comment by microbit-robert** [comment]
This looks good to me, pending the stubs changes being finalised.

**Comment by microbit-matt-hillsdon** [comment]
> This looks good to me, pending the stubs changes being finalised.

Thanks. As noted on the stubs issue, I'll merge this and track more niche stubs improvements separately.

**Pull request merged**